### DISCUSSION TITLE
CI/CD

### DISCUSSION THREAD

**Discussion by go-christianharrel** [PUP-BSIT/playground-group-vrzn_appdev#25] [Q&A] [answered]
What Is CI/CD and How Does It Work?

**Comment by go-christianharrel** [comment]
@PUP-BSIT/vrzn_appdev

**Comment by Naoe-Adrian** [comment, marked as answer]
CD (Continuous Deployment)
Where every change that passes through the automated testing pipeline is deployed to production automatically, without manual intervention. Example here are the netlify and hostinger. When we deploy our personal portfolio and host our project in SIA. 

CI (Continuous Integration) 
focuses on automating the integration of code changes from multiple contributors into a shared repository frequently. Just like what we did last sem in our SIA. We utilize github to have centralize repository and apply synchronized coding changes.

**Reply by go-christianharrel** [reply]
Sounds Great!
